### PR TITLE
Add required services for populations in Norway and add metadata

### PR DESCRIPTION
**Pull request by Kpaubert**
Metadata was also altered to be more correct in the Mongodb DB.

So far only tested with mocking data.

Relates to #81

### CODE REVIEW

**Review by FredrikAugust — COMMENTED** [review]
Loogietoomie except for the `ConfigureAwait` warning.

**Comment by FredrikAugust** [comment]
And I would like to see some tests added, but it's understandable if you wish to do so in another PR.

File: BackendTests/UnitTests/MetAPIServiceTest.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
+using System.Net.Http;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Backend.API.Services;
+using BackendTests.MockHelpers;
+using Xunit;
+
+namespace BackendTests.UnitTests
+{
+    public class MetAPIServiceTest
```
Comment (fredrikbw) [inline BackendTests/UnitTests/MetAPIServiceTest.cs:10]: These tests are not concerned with the current PR, and should be in a separate one

File: BackendTests/BackendTests.csproj [hunk 2]
```diff
@@ -8,6 +8,7 @@
 
     <ItemGroup>
         <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.9.1" />
+        <PackageReference Include="Moq" Version="4.16.1" />
```
Comment (fredrikbw) [inline BackendTests/BackendTests.csproj:11]: This can also be left out

**Review by fredrikbw — CHANGES_REQUESTED** [review]

File: BackendTests/UnitTests/MetAPIServiceTest.cs (hunk 1 above)
Comment (Kpaubert) [reply BackendTests/UnitTests/MetAPIServiceTest.cs:10]: Yes sir.

File: BackendTests/BackendTests.csproj (hunk 2 above)
Comment (Kpaubert) [reply BackendTests/BackendTests.csproj:11]: That's a negative.

**Review by fredrikbw — APPROVED** [review]
LGTM

**Pull request merged**